### ISSUE TITLE
Clarify semantic of top level user paths

### ISSUE THREAD

**Issue by msiglreith** [KhronosGroup/OpenXR-Docs#29]
`xrGetCurrentInteractionProfile` takes a `topLevelUserPath` argument, which is not totally clear to me how it should look like in practice. My initial assumption would be a `/user path` (6.3.1) but it also states it needs to follow the device input subpath syntax, which starts to confuse me as the semantic of 'top level' is unknown to me in this context.

**Comment by yl-msft** [comment]
I agree with you the "top level user path" has been showing in the spec many times without precise definition.  It is indeed the `/user path` in 6.3.1, and it's called "top level" when they are used without input/output subpath.
